pkg/apis/config/v1beta1/defaults.go

```diff
@@ -50,11 +50,11 @@ func SetDefaults_CoschedulingArgs(obj *CoschedulingArgs) {
 }
```

Comment:
If we change the args from point to int, `SetDefaults_CoschedulingArgs` also needs to be modified here.

Reply:
We don't need to, do we? As in `SetDefaults_CoschedulingArgs` is in versioned pkg, the argument (versioned args) is still a pointer.

Reply:
Got it. Thanks.  please rebase the code.

Reply:
Rebased.